README.md

```diff
@@ -23,7 +23,7 @@ Additionally, page objects for administrative forms are present in _circ_admin_p
 ## Configuration
 ### Environment variables
 
-Test scripts are run against the teardown QA server at https://disposable-circulation.librarysimplified.org/
+Test scripts are run against the teardown QA server at https://qa-circulation.librarysimplified.org/
```

Comment:
Is this right? qa-circulation is the regular QA server, not the teardown server.

Reply:
That is wrong. Let me fix that.